### DISCUSSION TITLE
I've received student approval, I'm unable to access the service.

### DISCUSSION THREAD

**Discussion by jihyunlee2023** [community/community#114564] [GitHub Education]
### Hi everyone,

I can't resolve this issue. Although I've received student approval, I'm unable to access the service. I haven't received any emails either.
<img width="315" alt="스크린샷 2024-03-28 오후 1 25 54" src="https://github.com/community/community/assets/152181713/6b557393-8c5f-4398-8717-045d2ff3836b">

**Comment by maxiuw** [comment]
same situation here... how long does it take for the benefits to be activated/accessible?

**Comment by RkBeau** [comment]
I am also in this exact situation.  I have been approved but no email or access.

**Comment by Huangyiqiang1203** [comment]
Your school email may have blocked emails. Try to check the email blacklist

**Comment by RkBeau** [comment]
I have gotten other emails from github
________________________________

**Reply by Huangyiqiang1203** [reply]
![0b6cc772fc961056ac615cf28efedf3](https://github.com/community/community/assets/155220539/c7b73b1d-b45b-470d-92ca-b8aa83f7d473)
Github has many senders, even if you can receive some of them, itdoesn't mean you can receive all of them.

**Comment by zhangyueyue12** [comment]
我也是不行，同样approve
